kratos/python/add_modeler_to_python.cpp

```diff
@@ -116,6 +117,20 @@ void  AddModelerToPython(pybind11::module& m)
     py::class_<VoxelMeshGeneratorModeler, VoxelMeshGeneratorModeler::Pointer, Modeler>(m, "VoxelMeshGeneratorModeler")
         .def(py::init<Model &, Parameters>())
     ;
+
+    py::class_<CleanUpProblematicTrianglesModeler, CleanUpProblematicTrianglesModeler::Pointer, Modeler>(m, "CleanUpProblematicTrianglesModeler")
+        .def(py::init<Model&, Parameters>())
+        .def(py::init<ModelPart&, Parameters>())
+        .def(py::init<ModelPart&>())
+        .def_static("CleanUpProblematicGeometriesInMesh", &CleanUpProblematicTrianglesModeler::CleanUpProblematicGeometriesInMesh,
+          py::arg("rThisModelPart"),
+          py::arg("rEntityType") = "element",
+          py::arg("FirstNodeId") = 1,
+          py::arg("FirstElementId") = 1,
+          py::arg("FirstConditionId") = 1,
+          py::arg("AreaTolerance") = 1.0e-4,
+          "Clean up the problematic geometries (null area geometries) in the mesh.")
```

Comment:
Neither this one. Again, I'd promote a standarized usage through the constructor.